### ISSUE TITLE
Widget generator

### ISSUE THREAD

**Issue by ppmim** [efisoft-elt/pydevmgr_elt#1]
In the [documentation ](https://pydevmgr-elt.readthedocs.io/en/latest/pydevmgr_elt_manual.html#guis)is mentioned the method get_widget_factory() to create a device widget, but that method does not seem to exist anymore.

Great job done for this package !

**Comment by efisoft-elt** [comment]
Dam. 
Thanks José-Miguel. Indeed it was move to `pydevmgr_core_qt` I will include it in `pydevmgr_elt_qt` and update the doc. 

```
from pydevmgr_core_qt import get_widget_factory
```

**Comment by efisoft-elt** [comment]
Solved in just release V0.5.0 . 

```python
from pydevmgr_elt_qt import get_widget_factory
```

**Comment by efisoft-elt** [comment]
closed
